electron/app/window-manager.ts

```diff
@@ -171,6 +173,14 @@ export default class WindowManager {
          * https://github.com/maxogden/menubar
          */
         let icon = os.platform() === 'darwin' ? config.icon : config.iconBig;
+        
+        // const contextMenu = Menu.buildFromTemplate([
+        //     { label: 'Open', type: 'radio'},
+        // ]);
+        // tray = new Tray(icon);
+        // if (os.platform() === 'linux') {
+        //     tray.setContextMenu(contextMenu);
+        // }
```

Comment:
Assuming this would fix the empty menu that's there when clicking on the tray icon, any chance getting this working as well?

Reply:
![DevTime-ubuntu](https://user-images.githubusercontent.com/43229944/121315789-41da2180-c93b-11eb-87e1-0791c97604bb.png)

Reply:
Currently, we cannot directly show the app when we click the icon. That's an open issue for electron. So only thing we can do is to add context. And we can add more context in the popUp, such as status or something, but we should write the tray ourself instead of just use the "menubar"